src/utils.js

```diff
@@ -0,0 +1,143 @@
+const homedir = require('os').homedir();
+const path = require('path');
+const fs = require('fs');
+const assert = require('bsert');
+
+const nearAPI = require('near-api-js');
+const Web3 = require('web3');
+
+const CREDENTIALS_DIR = '.near-credentials';
+const PROJECT_KEY_DIR = './neardev';
+
+const DEFAULT_GAS = 1000000;
+
+async function setupNear(config) {
+    const deps = await createLocalKeyStore(config.networkId, config.keyPath);
+    if (config.keyPath) {
+        delete config.keyPath;
+    }
+    return nearAPI.connect({ networkId: config.networkId, nodeUrl: config.nearNodeUrl, deps });
+}
+
+async function setupEth(config) {
+    const web3 = await getWeb3(config);
+    web3.eth.defaultAccount = addSecretKey(web3, config.ethFromSecretKey);
+    config.ethFrom = web3.eth.defaultAccount;
+    return web3;
+}
+
+/**
+ * Setup connection to NEAR and Ethereum from given configuration.
+ * @param {Object} config Config object which defines nearNodeUrl/ethNodeUrl, networkId and more.
+ */
+async function setupEthNear(config) {
+    const near = await setupNear(config);
+    const web3 = await setupEth(config);
+    return { near, web3 }
+}
+
+/**
+ * Remove 0x if prepended
+ * @param {String} value value to check and modify
+ * @return {String} string without 0x
+ */
```

Comment:
nice doc

Reply:
Hmm I think there's no `modify`. Assume it were, then 
```
    if (value.slice(0, 2) === '0x') {
        return value.slice(2);
```
wouldn't work, so value.slice returns new string and didn't modify exist

Reply:
Will be fixed in https://github.com/near/rainbow-bridge-lib/pull/13